app/models/notifications/also_commented.rb

```diff
@@ -18,7 +18,11 @@ def self.notify(comment, _recipient_user_ids)
       recipient_ids = commentable.participants.local.where.not(id: [commentable.author_id, actor.id]).pluck(:owner_id)
 
       User.where(id: recipient_ids).find_each do |recipient|
-        next if recipient.is_shareable_hidden?(commentable)
+        if recipient.is_shareable_hidden?(commentable) ||
+          Notifications::MentionedInComment.where(target: comment.mentions.where(person: recipient.person)).exists?
```

Comment:
`mention_notification_exists?` is going to be used both in this and `CommentOnPost` classes. Is it fine to place this method to the `Notification` model? I'd place it to `Notifications::MentionedInComment` as class-method, however it wouldn't be one-liner then (I checked, it doesn't fit).